cypress/e2e/room_board/collabora_files_rooms_board/addEditDeleteCollaboraTextFileInRoomBoard.feature

```diff
@@ -80,16 +80,16 @@ Feature: Room Board - Collabora full text file access (add, edit, delete) with e
         When I download by clicking on the option PDF download in collabora editor
         When I arrive on the dashboard
         When I go to rooms overview
-        When I go to room '<room_name>'
+        When I click on button Open to go to room '<room_name>' at position '0'
         When I click on the button Open on multi-column board in the room detail page
         Then I see the file type DOCX is uploaded in the card
 
         # first teacher deletes the collabora docx document from room board
         Given I am logged in as a '<teacher_1>' at '<namespace>'
-        Given the card file is deleted from room '<room_name>'
+        Given the card file is deleted from room '<room_name>' at position '0'
 
         # post-condition: delete the room
-        Given the room named '<room_name>' is deleted
+        Given the room named '<room_name>' at position '0' is deleted
```

Comment:
Also here a bit shorter:

Given the room '<room_name>' at position '0' is deleted

Reply:
same, but updated it.